### PR TITLE
Added Symfony 2.6 dependency to composer, updated readme, added .gitignore

### PR DESCRIPTION
**Pull request by clash82**
Ready for `v0.2` release.

### CODE REVIEW

**Comment by clash82** [comment]
ping @andrerom @bdunogier

**Comment by bdunogier** [comment]
Looks good. Wish it was tested by Travis, but we can't right now :-)

**Comment by clash82** [comment]
> Wish it was tested by Travis, but we can't right now :-)

I'll add it... later :-)

File: README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ This bundle adds privacy cookie banner into Symfony 2 application (eZ Publish /
 
 ## Requirements
 
-- any Symfony v2.x application
+- any Symfony v2.6 or newer application
```
Comment (andrerom) [inline README.md]: could mention v0.1 works with older versions

**Comment by andrerom** [comment]
+1

File: README.md (hunk 1 above)
Comment (clash82) [inline README.md]: Added.

**Comment by clash82** [comment]
Small update: I added short description about bundle abstraction.

**Pull request merged**